tutorials/applications/Simulating Sequences with Errors and Noise.ipynb

```diff
@@ -0,0 +1,1129 @@
+{
```

Comment:
<p>I'd suggest changing the last sentence of the first paragraph to <code>In order to realistically simulate these quantum computations, we need to take them into account.</code> ,</p><p>What do you think?</p><p> </p><p>Also, what about dephasing?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/232/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>